packages/flutter/test/material/search_bar_theme_test.dart

```diff
@@ -0,0 +1,318 @@
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('SearchBarThemeData copyWith, ==, hashCode basics', () {
+    expect(const SearchBarThemeData(), const SearchBarThemeData().copyWith());
+    expect(const SearchBarThemeData().hashCode, const SearchBarThemeData()
+        .copyWith()
+        .hashCode);
+  });
+
+  test('SearchBarThemeData lerp special cases', () {
+    expect(SearchBarThemeData.lerp(null, null, 0), null);
+    const SearchBarThemeData data = SearchBarThemeData();
+    expect(identical(SearchBarThemeData.lerp(data, data, 0.5), data), true);
+  });
+
+  test('SearchBarThemeData defaults', () {
+    const SearchBarThemeData themeData = SearchBarThemeData();
+    expect(themeData.elevation, null);
+    expect(themeData.backgroundColor, null);
+    expect(themeData.shadowColor, null);
+    expect(themeData.surfaceTintColor, null);
+    expect(themeData.overlayColor, null);
+    expect(themeData.side, null);
+    expect(themeData.shape, null);
+    expect(themeData.padding, null);
+    expect(themeData.textStyle, null);
+    expect(themeData.hintStyle, null);
+    expect(themeData.constraints, null);
+
+    const SearchBarTheme theme = SearchBarTheme(
+        data: SearchBarThemeData(), child: SizedBox());
+    expect(theme.data.elevation, null);
+    expect(theme.data.backgroundColor, null);
+    expect(theme.data.shadowColor, null);
+    expect(theme.data.surfaceTintColor, null);
+    expect(theme.data.overlayColor, null);
+    expect(theme.data.side, null);
+    expect(theme.data.shape, null);
+    expect(theme.data.padding, null);
+    expect(theme.data.textStyle, null);
+    expect(theme.data.hintStyle, null);
+    expect(theme.data.constraints, null);
+  });
+
+  testWidgets('Default SearchBarThemeData debugFillProperties', (
+      WidgetTester tester) async {
+    final DiagnosticPropertiesBuilder builder = DiagnosticPropertiesBuilder();
+    const SearchBarThemeData().debugFillProperties(builder);
+
+    final List<String> description = builder.properties
+        .where((DiagnosticsNode node) => !node.isFiltered(DiagnosticLevel.info))
+        .map((DiagnosticsNode node) => node.toString())
+        .toList();
+
+    expect(description, <String>[]);
+  });
+
+  testWidgets('SearchBarThemeData implements debugFillProperties', (
+      WidgetTester tester) async {
+    final DiagnosticPropertiesBuilder builder = DiagnosticPropertiesBuilder();
+    const SearchBarThemeData(
+      elevation: MaterialStatePropertyAll<double>(3.0),
+      backgroundColor: MaterialStatePropertyAll<Color>(Color(0xfffffff1)),
+      shadowColor: MaterialStatePropertyAll<Color>(Color(0xfffffff2)),
+      surfaceTintColor: MaterialStatePropertyAll<Color>(Color(0xfffffff3)),
+      overlayColor: MaterialStatePropertyAll<Color>(Color(0xfffffff4)),
+      side: MaterialStatePropertyAll<BorderSide>(
+          BorderSide(width: 2.0, color: Color(0xfffffff5))),
```

Comment:
this can be on one line

Reply:
Fixed this and others! I think the IDE did this kind of format automatically. Sorry for not catching this.